### PR TITLE
MAINT: rename default branch to main

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Updates references to default branch as `main`.

## Type of change

- This change is a documentation update

# How Has This Been Tested?

via travis

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
-# The master toctree document.
+# The main toctree document (using required variable name).
```
Comment (jklenzing) [inline docs/conf.py:65]: Good call.  will update other packages accordingly

**Comment by rstoneback** [comment]
Same test failures here as in the develop branch.

**Comment by jklenzing** [comment]
going to run some local tests to track this down

**Comment by jklenzing** [comment]
Working locally.  Also working here.

**Comment by jklenzing** [comment]
Found a docstring in the tests that was duplicated.  Fixed for clarity.

**Comment by rstoneback** [comment]
Release text wiki page exists:

https://github.com/pysat/pysat/wiki/v2.2.2-Release-Text

**Comment by rstoneback** [comment]
I ran into a bug on the ICON IVM processing. I try to keep a subset of Metadata via the .keep() method. Internally it calls .drop,  which runs into the Meta object attributes are immutable Error. The flag needs to be set within that method.

**Comment by jklenzing** [comment]
I'm done tweaking this branch, although we might add that fix here @rstoneback to minimize pulls

**Comment by jklenzing** [comment]
@rstoneback, is this ready to go?

**Comment by rstoneback** [comment]
I'm running the code through ICON once more from beginning to end to ensure everything is working.

**Comment by rstoneback** [comment]
> I'm running the code through ICON once more from beginning to end to ensure everything is working.

ICON run finished and geophysical parameters are the same across pysat v2.1 and v2.2.2 .
Good to go on my end.

**Comment by jklenzing** [comment]
Things left to do for the migration to `main`
- [ ] update default branch in settings
- [ ] sync with coveralls to update there
- [.] activate in readthedocs and updated default setting there

@rstoneback, I don't have access to the settings here or at RTD.  Could you set me up as a maintainer?  Or, conversely, I can point you to the switches to flip.

**Comment by rstoneback** [comment]
> 
> @rstoneback, I don't have access to the settings here or at RTD. Could you set me up as a maintainer? 

Yep. We/I should get these sorted out across the pysat family.

**Comment by rstoneback** [comment]
@jklenzing Friendly check-in before I head to the dentist and I'm out of commission for a bit. Do you need anything else from me on this pull?

**Comment by jklenzing** [comment]
I've got some access to settings, but not over the default branch.  I think I need to be bumped up in privileges.  I also need access in readthedocs

**Comment by rstoneback** [comment]
> I've got some access to settings, but not over the default branch. I think I need to be bumped up in privileges. I also need access in readthedocs

I changed you to maintainer earlier. Do I need to bump it up again? I'll poke at RTD.

**Comment by rstoneback** [comment]
> >  I also need access in readthedocs
> 

Can you send your RTD username?

**Comment by jklenzing** [comment]
jklenzing, same as github

**Comment by rstoneback** [comment]
> jklenzing, same as github

Thanks. Making sure access goes the right way. :)
You are listed as a maintainer.

**Comment by jklenzing** [comment]
looking over the settings in pysatModels, I need admin access here.  RTD just has maintainer. sorry for the confusion.

**Comment by rstoneback** [comment]
core folks are admin

**Comment by jklenzing** [comment]
Thanks!  I should have everything I need to finish this

**Pull request merged**